protocol/testutil/keeper/perpetuals.go

```diff
@@ -285,6 +289,7 @@ func CreateTestPricesAndPerpetualMarkets(
 			perp.Params.AtomicResolution,
 			perp.Params.DefaultFundingPpm,
 			perp.Params.LiquidityTier,
+			types.PerpetualMarketType_PERPETUAL_MARKET_TYPE_CROSS,
```

Comment:
Shouldn't this be pulled from the `perpetuals` passed in rather than being hard-coded?